doc/gtsam.lyx

```diff
@@ -1599,11 +1604,11 @@ q_{y}
 \begin_inset Formula $q=\left(q_{x},q_{y},q_{\theta}\right)$
```

Comment:
Should matrix representation be used here to avoid confusion? This definition of q seems inconsistent with q in the equation at line 1710

Reply:
See my comment above. I understand it's confusing if you want this document to be a primer on SE(2) and the fact that they can be represented as 3x3 matrices, but the audience here is (perhaps wrongly?) assumed knowledgeable about it.